src/lib.ts

```diff
@@ -138,50 +141,89 @@ function acceptBestGuzzlrQuest() {
   }
 }
 
-export function prepWandererZone(): Location {
+export enum draggableFight {
+  BACKUP,
+  WANDERER,
+}
+
+export function prepWandererZone(type: draggableFight = draggableFight.WANDERER): Location {
```

Comment:
Can we rename this like "pickWandererZoneAndPrep"? Current name is a little unclear

Reply:
aye aye